modules/local/resolve_found_neoepitopes.nf

```diff
@@ -1,23 +1,11 @@
-// Import generic module functions
-include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
-
-params.options = [:]
-options        = initOptions(params.options)
-
 process RESOLVE_FOUND_NEOEPITOPES {
```

Comment:
Maybe good to rename the process to RESOLVE_FOUND_CLASS1_NEOEPITOPES to name accordingly to the process RESOLVE_FOUND_CLASS2_NEOEPITOPES

Reply:
Good find, working on it already!